pkg/tcpip/iptables/targets.go

```diff
@@ -65,3 +66,26 @@ type ReturnTarget struct{}
 func (ReturnTarget) Action(tcpip.PacketBuffer) (RuleVerdict, string) {
 	return RuleReturn, ""
 }
+
+// RedirectTarget redirects the packet by modifying the destination port/IP.
+type RedirectTarget struct {
+	RangeSize uint32
+	Flags     uint32
+	MinIP     tcpip.Address
+	MaxIP     tcpip.Address
+	MinPort   uint16
+	MaxPort   uint16
```

Comment:
Redirect allows redirecting to range of ports, instead of one port. MinPort and MaxPort are used to specify port range. MinIP and MaxIP are used for IP range. Added a TODO.